custom_components/google_home/types.py

```diff
@@ -1,18 +1,5 @@
 """Various types used in type hints."""
-from typing import Iterable, List, Optional, Set, Tuple, TypedDict
-
-from typing_extensions import Protocol
-
-from homeassistant.helpers.entity import Entity
-
-
-class AddEntitiesCallback(Protocol):
```

Comment:
I realised we don't really need such precise type and simple `Callable` is fine for our case.